### PR TITLE
Add initial support for displaying permissions for users and groups

### PR DESCRIPTION
**Pull request by odinuge**

### CODE REVIEW

File: lego/apps/users/models.py [hunk 1]
```diff
@@ -222,6 +228,20 @@ def past_memberships(self):
             user=self, abakus_group__deleted=False
         ).select_related("abakus_group")
 
+    @abakus_cached_property
+    def nested_permissions(self):
```
Comment (odinuge) [inline lego/apps/users/models.py]: Could probably do this cleaner, but this is a good first try :tm:

**Comment by odinuge** [comment]
Agree @SmithPeder!

Did a small rewrite now, so you can take a look.

**Comment by SmithPeder** [comment]
Nice, those names made a bit more sense 🥇

**Comment by odinuge** [comment]
Ill give this a try in master :tm:

**Pull request merged**